lib-py/casmutils/xtal/symmetry.py

```diff
@@ -33,7 +38,49 @@ def make_factor_group(structure, tol):
     list(cu.sym.CartOp)
 
     """
-    return [CartOp(op) for op in _xtal.make_factor_group(structure._pybind_value,tol)]
+    return [CartOp(op) for op in _xtal.make_factor_group(structure._pybind_value, tol)]
+
+
+def make_perm_rep_factor_group(structure, tol):
```

Comment:
I will always complain about abbreviations, especially on anything facing the public. `make_factor_group_permutations` would be preferable, but you can probably come up with something better.

Nice documentation though, I like the example.